From c16717bc87eff8fe67fd2131730cab63fd80882b Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Mon, 16 Jan 2023 16:59:46 +0100
Subject: [PATCH 09/11] edk2-firmware-tegra: Disable nvdisp init for tegra194

The nvdispinit binary generated in Yocto fails to
boot, while the one generated during flashing works
as expected. Therefore, we will bundle the one
that comes in the NVidia BSP.

Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 .../recipes-bsp/uefi/edk2-firmware-tegra_35.1.0.bbappend         | 1 +
 1 file changed, 1 insertion(+)

diff --git a/layers/meta-balena-jetson/recipes-bsp/uefi/edk2-firmware-tegra_35.1.0.bbappend b/layers/meta-balena-jetson/recipes-bsp/uefi/edk2-firmware-tegra_35.1.0.bbappend
index ccfd504..6a6adda 100644
--- a/layers/meta-balena-jetson/recipes-bsp/uefi/edk2-firmware-tegra_35.1.0.bbappend
+++ b/layers/meta-balena-jetson/recipes-bsp/uefi/edk2-firmware-tegra_35.1.0.bbappend
@@ -25,3 +25,4 @@ do_deploy:append() {
      cp ${WORKDIR}/build/images/BOOTAA64.efi ${DEPLOYDIR}/bootfiles/EFI/BOOT/BOOTAA64.efi
 }
 
+NVDISPLAY_INIT_DEFAULT:tegra194=""
-- 
2.37.2

